src/components/ha-button-toggle-group.ts

```diff
@@ -7,30 +7,33 @@ import {
   CSSResult,
   css,
 } from "lit-element";
-
-import "./ha-icon-button";
+import { classMap } from "lit-html/directives/class-map";
 
 import { fireEvent } from "../common/dom/fire_event";
 import type { ToggleButton } from "../types";
 
+import "./ha-svg-icon";
+
 @customElement("ha-button-toggle-group")
 export class HaButtonToggleGroup extends LitElement {
-  @property() public buttons!: ToggleButton[];
+  @property({ attribute: false }) public buttons!: ToggleButton[];
 
   @property() public active?: string;
 
   protected render(): TemplateResult {
     return html`
-      <div>
+      <div class=${classMap({ single: this.buttons.length === 1 })}>
         ${this.buttons.map(
-          (button) => html` <ha-icon-button
-            .label=${button.label}
-            .icon=${button.icon}
-            .value=${button.value}
-            ?active=${this.active === button.value}
-            @click=${this._handleClick}
-          >
-          </ha-icon-button>`
+          (button) => html`
+            <ha-svg-icon
```

Comment:
Should these not be wrapped in a `<mwc-icon-button>`?

Reply:
Probably. I was confused on exactly what to do when not using the ha-button. but that makes more sense 😅